src/python/ensembl/production/hive/ProductionDBCopy.py

```diff
@@ -11,51 +11,85 @@
    See the License for the specific language governing permissions and
    limitations under the License.
 """
+import collections
+import json
 import time
 from urllib.parse import urlparse
 
 from ensembl.hive.HiveRESTClient import HiveRESTClient
 
+JobProgress = collections.namedtuple("JobProgress", "status_msg table_copied total_tables progress runtime")
```

Comment:
Where is this `namedtuple` used?

Reply:
Oups, remaining of a previous code. Thanks for spotting it! Obsolete now.

Reply:
No problem :) I guess `import collections` (line 14) can go too.